app/server/controller/src/main/java/io/syndesis/server/controller/integration/online/PublishHandler.java

```diff
@@ -131,13 +131,17 @@ private DeploymentData createDeploymentData(Integration integration, Integration
 
         String username = integrationDeployment.getUserId().orElseThrow(() -> new IllegalStateException("Couldn't find the user of the integration"));
 
+        String integrationId = integrationDeployment.getIntegrationId().orElseThrow(() -> new IllegalStateException("IntegrationDeployment should have an integrationId"));
+        String version = Integer.toString(integrationDeployment.getVersion());
         return DeploymentData.builder()
             .withVersion(integrationDeployment.getVersion())
-            .addLabel(OpenShiftService.INTEGRATION_ID_LABEL, Labels.sanitize(integrationDeployment.getIntegrationId().orElseThrow(() -> new IllegalStateException("IntegrationDeployment should have an integrationId"))))
-            .addLabel(OpenShiftService.DEPLOYMENT_ID_LABEL,  Integer.toString(integrationDeployment.getVersion()))
-            .addLabel(OpenShiftService.DEPLOYMENT_VERSION_LABEL, Integer.toString(integrationDeployment.getVersion()))
+            .addLabel(OpenShiftService.INTEGRATION_ID_LABEL, Labels.sanitize(integrationId))
```

Comment:
We don't need to sanitize on the id anymore because of #1869 , so we should just skip the call here.